exporters/jaeger/src/test/java/io/opentelemetry/exporter/jaeger/JaegerGrpcSpanExporterTest.java

```diff
@@ -288,16 +288,24 @@ private static void verifyBatch(Model.Batch batch) throws Exception {
     return tags.stream().filter(kv -> kv.getKey().equals(tagKey)).findFirst();
   }
 
-  @Test
-  @SuppressWarnings("PreferJavaTimeOverload")
-  void validConfig() {
+  void validTrustedConfig() {
```

Comment:
Looks like you accidentally deleted the `@Test` annotation along with the `@SuppressWarnings`

Reply:
added back